### DISCUSSION TITLE
Wazuh with log collector

### DISCUSSION THREAD

**Discussion by CISO-GYM** [wazuh/wazuh#23511] [Q&A]
i want to install agents on endpoints, and i have the manager on a remote site, what's the best way to collect logs and send them to the remote site?

**Comment by JackW6809** [comment]
The remote site where Wazuh-Manager is located. Is it exposed to the public internet? If so, then just install the agent on the endpoint and use either the FQDN of the server or just the IP address to point to the manager. If not you will have to have a proxy solution or VPN solution so they can send the logs back really.

**Reply by CISO-GYM** [reply]
Yes it is exposed to the internet, so i can make a vpn connection, but i can't install agents on all endpoints i just want to collect logs from devices and forward them to my manager

**Reply by styxiner** [reply]
If you only want to collect logs without agents on some endpoints, you could try the agentless monitoring:
https://documentation.wazuh.com/current/user-manual/capabilities/agentless-monitoring/how-it-works.html
